packages/syft/src/syft/core/node/common/client_manager/dataset_api.py

```diff
@@ -123,6 +123,42 @@ def __init__(self, client: AbstractNodeClient):
     def create_syft(self, **kwargs: Any) -> None:
         super().create(**kwargs)
 
+    def delete(self, dataset_id: str, skip_checks: bool = False) -> None:  # type: ignore
+
+        if not skip_checks:
+            pref = input(
+                f"You are about to delete the dataset with id `{dataset_id}`? 🚨 \n"
+                "All information related to this dataset will be permanently deleted.\n"
+                "Please enter y/n to proceed: "
+            )
+            while pref != "y" and pref != "n":
+                pref = input("Invalid input '" + pref + "', please specify 'y' or 'n'.")
+            if pref == "n":
+                sys.stdout.write("Dataset deletion is cancelled.")
+                return
+
+        super().delete(dataset_id=dataset_id)
+        sys.stdout.write("Dataset is successfully deleted. ✅")
+
+    def del_asset(
+        self, dataset_id: str, asset_id: str, skip_checks: bool = False
+    ) -> None:
+
+        if not skip_checks:
+            pref = input(
+                f"You are about to delete asset with id: {asset_id} in the dataset with id `{dataset_id}`? 🚨 \n"
+                "All information related to this asset will be permanently deleted.\n"
+                "Please enter y/n to proceed: "
+            )
+            while pref != "y" and pref != "n":
+                pref = input("Invalid input '" + pref + "', please specify 'y' or 'n'.")
```

Comment:
MAYBE: Use f-strings.

Reply:
Fixed in: https://github.com/OpenMined/PySyft/pull/7079/commits/c664423caee20d946a3d9adc0fc55121dbd5fcc7